guides/v2.2/install-gde/prereq/security.md

```diff
@@ -1 +1,66 @@
-../../../v2.1/install-gde/prereq/security.md
\ No newline at end of file
+---
+group: installation-guide
+subgroup: Prerequisites
+title: SELinux and iptables
+menu_title: SELinux and iptables
+menu_order: 200
+functional_areas:
+  - Install
+  - System
+  - Setup
+---
+
+## SELinux {#install-prereq-selinux}
+[Security Enhanced Linux (SELinux)](http://selinuxproject.org/page/Main_Page){:target="_blank"} enables CentOS and Ubuntu administrators greater access control over their servers. If you're using SELinux _and_ Apache to initiate a connection to another host, you must run the commands discussed in this section.
+
+{:.bs-callout .bs-callout-info}
+  Magento has no recommendation about using SELinux; you can use it for enhanced security if you wish. If you use SELinux, you must configure it properly or the Magento application will function unpredictably. If you choose to use SELinux, consult a resource like [the CentOS wiki](http://wiki.centos.org/HowTos/SELinux){:target="_blank"} to set up rules to enable communication.
+
+### Suggestion for installing the Magento software with Apache
+
+If you choose to enable SELinux, you might have issues running the installer unless you change the *security context* of some directories as follows:
+
+	chcon -R --type httpd_sys_rw_content_t <your Magento install dir>/app/etc
+	chcon -R --type httpd_sys_rw_content_t <your Magento install dir>/var
+	chcon -R --type httpd_sys_rw_content_t <your Magento install dir>/pub/media
+	chcon -R --type httpd_sys_rw_content_t <your Magento install dir>/pub/static
+  chcon -R --type httpd_sys_rw_content_t <your Magento install dir>/generated
```

Comment:
The new line that you added for the `generated/` directory needs to be indented to match the preceding lines